### DISCUSSION TITLE
Configuring Transcription to Spanish Language

### DISCUSSION THREAD

**Discussion by pghb15** [amazon-connect/voicemail-express-amazon-connect#19] [Q&A] [answered]
Hey all! 
I was wondering if anyone could guide me into modifying my voicemail express configuration to change language of transcription from English to Spanish. Maybe changing a lambda function. Thanks in advance! PH

**Comment by pghb15** [comment, marked as answer]
Just realized theres a set contact attribute block within the test contact flow that is called "Set remaining required attributes for VMX3". There you can modify transcription language. Cool!

**Reply by dougjaso** [reply]
This is also [discussed here](https://github.com/amazon-connect/voicemail-express-amazon-connect/blob/main/Docs/vmx_core.md#required-contact-attributes), As long as the attribute is set to a language that Transcribe understands, it will work just fine.
